docs/reference/mapping/explicit-mapping.asciidoc

```diff
@@ -0,0 +1,148 @@
+
+[[explicit-mapping]]
+== Explicit mapping
+
+You know more about your data than {es} can guess, so while dynamic
+mapping can be useful to get started, at some point you will want to specify
+your own explicit mappings.
+
+You can create field mappings when you <<create-mapping,create an index>> and
+<<add-field-mapping,add fields to an existing index>>.
+
+[discrete]
+[[create-mapping]]
+=== Create an index with an explicit mapping
```

Comment:
runtime fields are not mentioned at all, maybe the examples should have a mix of runtime fields and indexed fields? Or should we relegate runtime fields to their own page? I am torn on this

Reply:
I think that we can explore this distinction in a separate PR. It's a valuable conversation.

Reply:
++